### PR TITLE
#1228  - Sync Documentation

### PR DESCRIPTION
**Pull request by lianplass**
Documentation for Sync property alongside comments.  Unsure if path in comment is correct, but happy to correct if otherwise.

### CODE REVIEW

**Review by matteobruni — CHANGES_REQUESTED** [review]
Good job but I cannot merge it because it's not following the standard of other files.

You can see here the `IAnimatableColor` interface: https://github.com/matteobruni/tsparticles/blob/69156ca7abe5684d4353c2d1a60f5a789a7e881a/engine/src/Options/Interfaces/IAnimatableColor.ts#L7

It has a markdown file describing the interface instead of a single property. This is the standard followed everywhere.

So the `sync` property should be described everywhere where used, with all other interface/class properties.

**Comment by lianplass** [comment]
Took another stab at it (hope I got it right this time!).  Once confirmed, i'll proceed with the others!

File: engine/src/Options/Classes/AnimationOptions.ts [hunk 1]
```diff
@@ -2,6 +2,11 @@ import { RecursivePartial } from "../../Types";
 import { IOptionLoader } from "../Interfaces/IOptionLoader";
 import { IAnimation } from "../Interfaces/IAnimation";
 
+/**
+ * [[include:engine\markdown\Sync.md]]
```
Comment (matteobruni) [inline engine/src/Options/Classes/AnimationOptions.ts]: This line needs to be changed

File: engine/src/Options/Classes/Particles/Life/Life.ts [hunk 2]
```diff
@@ -4,6 +4,11 @@ import type { RecursivePartial } from "../../../../Types";
 import { LifeDelay } from "./LifeDelay";
 import { LifeDuration } from "./LifeDuration";
 
+/**
+ * [[include:engine\markdown\Sync.md]]
```
Comment (matteobruni) [inline engine/src/Options/Classes/Particles/Life/Life.ts]: This line needs to be changed

**Review by matteobruni — CHANGES_REQUESTED** [review]
Good job, everything's better now, some files still needs a review.

**Review by matteobruni — APPROVED** [review]
Everything looks good. Good job!

**Pull request merged**